ClimbingPlaylistApi/Models/PlaylistModel.cs

```diff
@@ -12,22 +13,16 @@ namespace ClimbingPlaylistApi.Models
     /// </summary>
     public class PlaylistModel
     {
-        public PlaylistModel(string name = "New List")
-        {
-            Name = name;
-            Routes = new List<RouteModel>();
-            CreationDate = DateTime.Now;
-        }
-
         [Key]
         public int Id { get; init; }
-        
-        [MaxLength(256)]
-        public string Name { get; set; }
 
         [Required]
-        public DateTime CreationDate { get; set; }
+        [MaxLength(256)]
+        public required string Name { get; set; }
+
+        //[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public DateTime CreationDate { get; set; } = DateTime.Now;
         
-        public List<RouteModel> Routes { get; set; }
+        public List<RouteModel> Routes { get; set; } = new List<RouteModel>();
```

Comment:
I stopped doing this in favor of `public List<RouteModel>? Routes { get; set; }`